doc/generic/pgf/color.cfg

```diff
@@ -0,0 +1,5 @@
+% xcolor inputs this file and ignores the global document options to choose the engine
+% Here we just make sure that a driver is selected
+\expandafter\ifx\csname Gin@driver\endcsname\relax
+  \errmessage{No graphics driver has been selected. Load graphicx before xcolor.}%
+\fi
```

Comment:
Is this default `color.cfg` still needed? Both `color` and `xcolor` packages have released new versions containing fixing for latex3/xcolor/issues/6 (assume that's why this `color.cfg` ever existed).

Reply:
Let's leave it there for now, because TeX Live 2021 is not frozen yet.